### PR TITLE
[JENKINS-38987 Follow-up] Pick up SCMxxx.getPronoun()

### PR DESCRIPTION
**Pull request by stephenc**
@reviewbybees

### CODE REVIEW

**Comment by jglick** [comment]
Downstream of something?

File: src/main/java/jenkins/branch/MultiBranchProject.java [hunk 1]
```diff
@@ -143,6 +144,21 @@ private synchronized void init2() {
...
+    @Override
+    public String getPronoun() {
+        Set<String> result = new TreeSet<>();
+        for (BranchSource source: sources) {
+            String pronoun = Util.fixEmptyAndTrim(source.getSource().getPronoun());
+            if (pronoun != null) {
+                result.add(pronoun);
+            }
+        }
+        return result.isEmpty() ? super.getPronoun() : StringUtils.join(result, "/");
```
Comment (jglick) [inline src/main/java/jenkins/branch/MultiBranchProject.java]: Probably want spaces around the slash at least.

File: src/main/java/jenkins/branch/OrganizationFolder.java [hunk 2]
```diff
@@ -210,6 +223,21 @@ public void addAttribute(String key, Object value) throws IllegalArgumentExcepti
...
+    @Override
+    public String getPronoun() {
+        Set<String> result = new TreeSet<>();
+        for (SCMNavigator navigator: navigators) {
+            String pronoun = Util.fixEmptyAndTrim(navigator.getPronoun());
+            if (pronoun != null) {
+                result.add(pronoun);
+            }
+        }
+        return result.isEmpty() ? super.getPronoun() : StringUtils.join(result, "/");
```
Comment (jglick) [inline src/main/java/jenkins/branch/OrganizationFolder.java]: ditto

**Comment by abayer** [comment]
:bee:

**Comment by stephenc** [comment]
@reviewbybees done

**Comment by stephenc** [comment]
Merging as maintainer

**Pull request merged**